frameworks/CSharp/aspnetcore/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -12,10 +13,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Version="2.1.3" />
+    <PackageReference Condition="'$(NET21)' == 'true'" Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Version="2.1.3" />
+    <PackageReference Condition="'$(NET21)' != 'true'" Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Version="2.2.0" />
     <PackageReference Include="Utf8Json" Version="1.3.7" />
-    <PackageReference Include="Npgsql" Version="4.0.3" />
-    <PackageReference Include="MySqlConnector" Version="0.47.1" />
+    <PackageReference Include="Npgsql" Version="4.0.4" />
+    <PackageReference Include="MySqlConnector" Version="0.48.2" />
     <PackageReference Include="RedHat.AspNetCore.Server.Kestrel.Transport.Linux" Version="2.1.0-*" />
```

Comment:
Update `RedHat.AspNetCore.Server.Kestrel.Transport.Linux` to Version `2.2.0-*` rather than keeping the .cspoj on 2.1?

/cc @tmds

Reply:
Totally missed that there is one :) Changed, thank you

Reply:
👍